index.js

```diff
@@ -1,22 +1,23 @@
 var express = require('express');
 var fs = require('fs');
-var http = require('http');
 // Create a service (the app object is just a callback).
 var app = express();
 
 //hashing service
 var crypto = require('crypto');
 
+let config = JSON.parse(fs.readFileSync('config.json'));
+
+// Default to production mode (for development mode set
+// development mode set it to 'development')
+// do not return full stacktrace to random people on internet
+//app.set('env', config.mode || 'production');
```

Comment:
Should this be commented out?

Reply:
I wasn't sure whether you wanted this, so I commented it out for compatibility purposes. Ideally, yes Express would run in production mode on production.